ppcp-gateway/class-angelleye-paypal-ppcp-payment.php

```diff
@@ -2909,7 +2926,7 @@ public function angelleye_ppcp_capture_order_using_payment_method_token($order_i
                             if (!empty($_POST['wc-angelleye_ppcp_cc-payment-token']) && $_POST['wc-angelleye_ppcp_cc-payment-token'] != 'new') {
                                 $token_id = wc_clean($_POST['wc-angelleye_ppcp_cc-payment-token']);
                             } else {
-                                $payment_tokens_id = get_post_meta($order_id, '_payment_tokens_id', true);
+                                $payment_tokens_id = angelleye_ppcp_get_post_meta($order, '_payment_tokens_id', true);
```

Comment:
Why are we using 2 different methods in different places, for ex: few places I see 
$order->get_meta() function usage, while here you are using angelleye_ppcp_get_post_meta(), is there any difference?